src/main/haskell/kore/src/Kore/IndexedModule/IndexedModule.hs

```diff
@@ -123,14 +126,60 @@ data IndexedModule param pat atts =
         -- ^ map from builtin domain (symbol and sort) identifiers to the hooked
         -- identifiers
     }
-    deriving (Functor, Generic, Show)
+    deriving (Generic, Show)
+
+instance Bifunctor (IndexedModule param pat) where
```

Comment:
Is this really a good idea? I mean, when someone is reading the code, why would it make sense that "first" and "second" for an indexed module would refer to a rather obscure part of it (i.e. one kind of attributes)?

I know that those are the last two arguments, and Bifunctor works that way, but does this actually make sense, or does it look more like a hack that saves a bit of work?

Assuming that my questions make some sense, then why not make it a Trifunctor? That way you're not making it a functor over the obscure things only, you're making it a functor over (almost) everything.

My questions are genuine, not rethorical.

Reply:
Well, I'm not sure.  I was just continuing existing work which made it a `Functor`.  On the other hand, the only usage is to replace the attributes by `Null`.  I guess it made more sense when it was just a `Functor` and the instance was automatically derived.

Reply:
I also found that bit weird even from before (that `IndexedModule` is a `Functor` for its attributes. Perhaps lenses would be a better choice here?

Reply:
I just went on an removed teh Bifunctor instance. If we ever need it for more than one function, we can revisit this issue.